lib/impl/AzureBlobFiles.js

```diff
@@ -383,6 +382,8 @@ class AzureBlobFiles extends Files {
    * @private
    */
   async _getAzureBlobPresignCredentials (params) {
+    if (this._azure.sasCreds) { logAndThrow(new codes.ERROR_UNSUPPORTED_OPERATION({ messageValues: ['generatePresignURL'], sdkDetails: params })) }
```

Comment:
I think the condition should be `this.hasOwnCredentials && this._azure.sasCreds` because tvm credentials ARE SAS credentials

Reply:
we already check for `this.hasOwnCredentials` within _getPresignUrl call, so we don't need to repeat that check.

Reply:
ok 👍